src/app/pages/component-sidenav/component-sidenav.ts

```diff
@@ -37,9 +37,12 @@ import {HttpClientModule} from '@angular/common/http';
 import {StackBlitzButtonModule} from '../../shared/stack-blitz';
 import {SvgViewerModule} from '../../shared/svg-viewer/svg-viewer';
 import {ExampleModule} from '@angular/components-examples';
+import {ComponentSidenavService} from './component-sidenav.service';
+import {MatDrawerToggleResult} from '@angular/material/sidenav/drawer';
 import {MatListModule} from '@angular/material/list';
 
-const SMALL_WIDTH_BREAKPOINT = 720;
+const EXTRA_SMALL_WIDTH_BREAKPOINT = 720;
```

Comment:
Add comment that explains how you arrived at these values?
(at least for the new one, not sure where the original 720 came from)